src/DefaultContainer.php

```diff
@@ -82,19 +86,23 @@ protected function handleError(Container $container, Request $request, \Throwabl
     {
         $this->logError($request, $exception);
 
+        $json = json_encode($this->getErrorResponse($exception));
+        $streamBody = fopen('data://text/plain,' . $json,'r');
         return $container["response"]
             ->withStatus($this->getStatusCode($exception))
-            ->withJson($this->getErrorResponse($exception))
+            ->withBody(new Stream($streamBody))
             ->withHeader("Access-Control-Allow-Origin", "*");
     }
 
-    public function __construct()
+    public function __construct(
+        InjectionFactory $injectionFactory,
+        ContainerInterface $delegateContainer = null)
```

Comment:
What do these added params do?

Reply:
I'm not sure how these objects are used. The Container class from Slim v3 was changed to use the Aura/DI/Container class in Slim v4 and now these are required params.